Src/library/base/elks/kernel/function.e

```diff
@@ -23,7 +23,8 @@ inherit
 		end
 
 create {NONE}
-	set_rout_disp
+	set_rout_disp,
+	set_rout_disp_final
```

Comment:
'set_rout_disp_final' is used in finalized mode to create agent objects from the runtime.